storage/storage-mq/src/storage-content/postgresStorageContentRepository.ts

```diff
@@ -41,10 +53,10 @@ export class PostgresStorageContentRepository implements StorageContentRepositor
   }
 
   async existsTable (tableIdentifier: string): Promise<boolean> {
-    const resultSet =
+    const resultSet: QueryResult<ExistsTableResultRow> =
       await this.postgresRepository.executeQuery(EXISTS_TABLE_STATEMENT(POSTGRES_SCHEMA, tableIdentifier), [])
-    const foundTableWithIdentifier = !!resultSet.rows[0].to_regclass
-    return foundTableWithIdentifier
+    // to_regclass contains the table name as string or null if it does not exists
```

Comment:
Was already present in the codebase, but probably would be cleaner to extract to method with a speaking name... would you mind including that in your PR?

Reply:
Already done